common/views.py

```diff
@@ -557,6 +558,18 @@ def delete(self, request, pk, format=None):
         return Response({"status": "success"}, status=status.HTTP_200_OK)
 
 
+# Utility function for currency conversion
+def convert_to_euros(amount, currency, conversion_rates):
+    try:
+        # Ensure amount is a float
+        amount = float(amount)
+    except (TypeError, ValueError):
+        # Handle cases where the amount is not a valid number
+        amount = 0
+    conversion_rate = conversion_rates.get(currency, 1.0)  # Default to 1.0 if not found
+    return amount * conversion_rate
```

Comment:
This should change to : 
```python
amount /conversion_rate
``` 

Also now it returns 
```powershell
  "total_revenue_in_euros": 50.877777
``` 

Let's round it up to 2 digits after the decimal point.